indexer/database/bridge_transfers.go

```diff
@@ -129,8 +141,11 @@ l1_transaction_deposits.l2_transaction_hash`)
 	depositsQuery = depositsQuery.Joins("INNER JOIN l1_transaction_deposits ON l1_bridge_deposits.transaction_source_hash = l1_transaction_deposits.source_hash")
 	depositsQuery = depositsQuery.Joins("INNER JOIN l1_contract_events ON l1_transaction_deposits.initiated_l1_event_guid = l1_contract_events.guid")
 
-	// add in cursoring options
-	filteredQuery := depositsQuery.Where(&Transaction{FromAddress: address}).Order("l1_bridge_deposits.timestamp DESC").Limit(100)
+	if cursor != "" {
```

Comment:
I think we need to make sure this deposit is ordered by timestamp to make sure this works right. Realized i forgot to add the ordering clause to these.

That way the cursor also doesn't have to be returned in the response. The caller simply knows that the last entry is the cursor to pass into the next page

Reply:
Will change it to timestamp in it's own pr but is block_number better?